runelite-client/src/main/java/net/runelite/client/game/ItemMapping.java

```diff
@@ -85,6 +85,14 @@
 	ITEM_DRAGON_BOOTS(DRAGON_BOOTS, DRAGON_BOOTS_G),
 	ITEM_DRAGON_BOOTS_ORNAMENT_KIT(DRAGON_BOOTS_ORNAMENT_KIT, DRAGON_BOOTS_G),
 
+	// Rune ornament kits
+	ITEM_RUNE_SCIMITAR_GUTHIX(RUNE_SCIMITAR, RUNE_SCIMITAR_23330),
+	ITEM_RUNE_SCIMITAR_ORNAMENTKIT_GUTHIX(RUNE_SCIMITAR_ORNAMENT_KIT_GUTHIX, RUNE_SCIMITAR_23330),
+	ITEM_RUNE_SCIMITAR_SARADOMIN(RUNE_SCIMITAR, RUNE_SCIMITAR_23332),
+	ITEM_RUNE_SCIMITAR_ORNAMENTKIT_SARADOMIN(RUNE_SCIMITAR_ORNAMENT_KIT_SARADOMIN, RUNE_SCIMITAR_23332),
+	ITEM_RUNE_SCIMITAR_ZAMORAK(RUNE_SCIMITAR, RUNE_SCIMITAR_23334),
+	ITEM_RUNE_SCIMITAR_ORNAMENTKIT_ZAMORAK(RUNE_SCIMITAR_ORNAMENT_KIT_ZAMORAK, RUNE_SCIMITAR_23334),
```

Comment:
Use `ORNAMENT_KIT` instead of `ORNAMENTKIT`

Reply:
Rune defender (t) should also be added under Rune ornament kits.
I believe the Tzhaar-ket-om (t) is also missing. I don't know where to put this under tho since it doesn't seem to fit under any comment. Could be placed under Miscellaneous ornament kits or a new comment for weaponry.

Reply:
I added the Rune Defender, Tzhaar-ket-om under misc., and fixed the `ORNAMENT_KIT` typos.